cfe_ts_crc.c

```diff
@@ -102,20 +102,21 @@ uint32 CalculateCRC(void *DataPtr, uint32 DataLength, uint32 InputCRC)
 
 int main(int argc, char **argv)
 {
-    int    readSize;
-    int    skipSize = 0;
-    int    fileSize = 0;
-    uint32 fileCRC  = 0;
-    int    fd;
-    int    done = 0;
-    char   buffer[100];
+    int     readSize;
+    int     skipSize = 0;
+    int     fileSize = 0;
```

Comment:
Since we opened the pandora's box of type issues here - we should fix these too:
 - `readSize` and `fileSize` should be `ssize_t`  (note, not size_t but ssize_t - posix type for signed size)
 - `skipSize` should be `off_t`